config/core/roles/webhook-clusterrole.yaml

```
@@ -109,6 +109,18 @@ rules:
       - "create"
       - "patch"
 
+  - apiGroups:
+      - ""
+    resources:
+      - "serviceaccounts"
+    verbs:
+      - "get"
+      - "create"
+      - "update"
+      - "list"
+      - "watch"
+      - "patch"
```

```suggestion
    verbs: *everything
```
nit: as we declared it in Line 75, we can use the short form here